$NetBSD: patch-configure,v 1.1 2014/12/30 15:45:05 wiz Exp $
* pkgsarc doesn't use lib64 so don't set it (fixes build on linux)
http://sf.net/p/qore/code/6817/
* fix typo so that --with-sqlite3 works correctly
http://sourceforge.net/p/qore/code/6348/
--- configure.orig	2010-06-20 09:09:23.000000000 +0000
+++ configure
@@ -14259,7 +14259,6 @@ if test "$enable_64bit" = "yes"; then
       *linux*)	if test "$GXX" = "yes"; then
 		   CXXFLAGS="$CXXFLAGS -m64"
 		fi
-		LIBSUFFIX=64
 
 		;;
       *hpux*)	if test "$host_cpu" = "ia64"; then
@@ -14717,7 +14716,7 @@ fi
 
 # Check whether --with-sqlite3 was given.
 if test "${with_sqlite3+set}" = set; then :
-  withval=$with_sqlite3; if test ! -d "${with_sqlite3l}"; then as_fn_error "directory ${with_sqlite3} does not exist for --with-sqlite3" "$LINENO" 5; unset with_sqlite3; fi
+  withval=$with_sqlite3; if test ! -d "${with_sqlite3}"; then as_fn_error "directory ${with_sqlite3} does not exist for --with-sqlite3" "$LINENO" 5; unset with_sqlite3; fi
 fi
 
 
